### PR TITLE
Discontinue support for tests running on Travis CI

### PR DESCRIPTION
**Pull request by sgiehl**
### Description:

Remove remaining occurrences of Travis across the code base.

refs #20179 

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by sgiehl** [comment]
The remaining failing UI tests should be random, our caused by submodules that were not yet fixed in their repo....
@bx80 most of the updated screenshots were actually already outdated on 5.x-dev. So the changes are not caused within this PR.

File: core/testMinimumPhpVersion.php [hunk 1]
```diff
@@ -19,7 +19,6 @@
 // Minimum requirement: stream_resolve_include_path, working json_encode in 5.3.3, namespaces in 5.3
 // NOTE: when changing this variable, we also need to update
 // 1) api.matomo.org
-// 2) tests/travis/generator/Generator.php
 // 3) composer.json (in two places)
 // 4) tests/PHPUnit/Integration/ReleaseCheckListTest.php
```
Comment (bx80) [inline core/testMinimumPhpVersion.php]:
```suggestion
// 2) composer.json (in two places)
// 3) tests/PHPUnit/Integration/ReleaseCheckListTest.php
```

File: tests/PHPUnit/Framework/Fixture.php [hunk 2]
```diff
@@ -374,8 +374,8 @@ public function markFixtureSetUp()
 
     public function performTearDown()
     {
-        // Note: avoid run SQL in the *tearDown() metohds because it randomly fails on Travis CI
-        // with error Error while sending QUERY packet. PID=XX
+        // Note: avoid run SQL in the *tearDown() metohds because it randomly fails on CI
```
Comment (bx80) [inline tests/PHPUnit/Framework/Fixture.php]:
```suggestion
        // Note: avoid run SQL in the *tearDown() methods because it randomly fails on CI
```

**Review by bx80 — APPROVED** [review]
Left comments for a couple of very minor text tweaks, but otherwise I can't see any issues with these changes. :+1: 

The system api service test is failing on other builds too so is unrelated.

* [x] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [x] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [x] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [x] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [x] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [x] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [x] n/a [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [x] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [x] Updated [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [x] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [x] Existing documentation updated if needed

**Pull request merged**